source/includes/aggregation/aggregation.php

```diff
@@ -36,5 +44,77 @@
 
 $result = $collection->explain($aggregate);
 echo json_encode($result), PHP_EOL;
-// end-explain
+// end-array-explain
+
+// start-builder-match-group
+$pipeline = [
+    Stage::match(
```

Comment:
Since these code excerpts are rendered as-is in the docs, the `use` statements at the top of this file won't be presented to users and it won't be clear exactly which classes are being referred to here.

Omitting the `use` statements and using the full class names here would address this at the expense of more verbose examples. Alternatively, you can find some way to include the `use` statements in the docs; however, that could be problematic since the examples in this file appear in separate sections -- and repeating them for each section might also be verbose.

Reply:
I'll use the fully qualified class name in the examples instead!